articles/active-directory-b2c/active-directory-b2c-reference-oidc.md

```diff
@@ -271,6 +271,7 @@ GET https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{policy}/oauth2/v2.0/
 | {tenant} | Yes | Name of your Azure AD B2C tenant |
 | {policy} | Yes | The user flow that you want to use to sign the user out of your application. |
 | id_token_hint| No | A previously issued ID token to pass to the logout endpoint as a hint about the end user's current authenticated session with the client. The `id_token_hint` ensures that the `post_logout_redirect_uri` is a registered reply URL in your Azure AD B2C application settings. |
+| client_id | No* | The application ID that the [Azure portal](https://portal.azure.com/) assigned to your application.<br>\**This is required when using `Application` isolation SSO Configuration* |
```

Comment:
```suggestion
| client_id | No* | The application ID that the [Azure portal](https://portal.azure.com/) assigned to your application.<br>\**This is required when using `Application` isolation SSO configuration and _Require ID Token_ in logout request is set to `No`.* |
```

Reply:
@eric-winkler Hi Eric, if you merge the suggested change we can get this signed off and published. If the change isn't accurate, feel free to correct as necessary. Thanks!